hbase-client/src/test/java/org/apache/hadoop/hbase/ipc/TestNettyRpcConnection.java

```diff
@@ -67,6 +68,7 @@ public static void tearDown() throws IOException {
     Closeables.close(CLIENT, true);
   }
 
+  @Ignore
```

Comment:
is there a new reason to ignore this test with this change?

Reply:
This test doesn't make sense anymore, because one of the main changes in this patch is to avoid running HBase methods running on the event loop. Previously I removed the test which caused unexpected issues with the CI and as @busbey suggested, I just Ignore this test for now and remove it in a later patch.

Reply:
were you planning to remove this test now?